utils/release_helper.py

```diff
@@ -0,0 +1,790 @@
+
+Welcome to the new contributors:
+{new_contributors}
+
+For full release notes, please have a look at:
+{latest_release_url}
+
+Pre-built content: {prebuild_zip}
+SHA-512 hash: {prebuild_zip_hash}
+
+Source tarball: {source_tarball}
+SHA-512 hash: {source_tarball_hash}
+
+Thank you to everyone who contributed!
+
+Regards,'''
+    return template
+
+
+def print_communication_channels(phase='release') -> None:
+    gitter_lnk = 'https://gitter.im/Compliance-As-Code-The/content'
+    ghd_lnk = 'https://github.com/ComplianceAsCode/content/discussions/new?category=announcements'
+    twitter_lnk = 'https://twitter.com/openscap'
+    ssg_mail = 'scap-security-guide@lists.fedorahosted.org'
+    openscap_mail = 'open-scap-list@redhat.com'
+
+    print('Please, share the following message in:\n'
+          f'* Gitter ({gitter_lnk})\n'
+          f'* Github Discussion ({ghd_lnk})\n'
+          f'* SCAP Security Guide Mail List ({ssg_mail})')
+
+    if phase == "finish":
+        print(f'* OpenSCAP Mail List ({openscap_mail})\n'
+              f'* Twitter ({twitter_lnk})')
+
+
+# Issues and PRs
+def filter_items_outdated_milestone(objects_list, milestone) -> list:
+    items_old_milestone = []
+    for item in objects_list:
+        if has_milestone_version_format(item.milestone) and \
+                item.milestone.title != milestone.title:
+            items_old_milestone.append(item)
+    return items_old_milestone
+
+
+def filter_outdated_items(repo, objects_list, type) -> list:
+    latest_milestone = get_latest_version_milestone(repo)
+    items_to_update = filter_items_outdated_milestone(objects_list, latest_milestone)
+    if items_to_update:
+        count = len(items_to_update)
+        print(f'The following {count} {type}(s) have an outdated milestone:')
+        for item in items_to_update:
+            print(f'{item.number:5} - {item.url:65} - {item.milestone.title} - {item.title}')
+        if get_confirmation(
+                f'Are you sure about updating the milestone in {count} Open {type}s?'):
+            return items_to_update
+        else:
+            print(f'Aborted! {type}(s) milestones not updated.')
+    else:
+        print(f'Great! There is no {type} with an outdated milestone.')
+        return []
+
+
+def get_items_with_milestone(object_list) -> list:
+    with_milestone = []
+    for item in object_list:
+        if item.milestone:
+            with_milestone.append(item)
+    return with_milestone
+
+
+def get_open_issues_with_milestone(repo) -> list:
+    open_issues = repo.get_issues(state='open', direction='asc')
+    return get_items_with_milestone(open_issues)
+
+
+def get_open_prs_with_milestone(repo) -> list:
+    open_prs = repo.get_pulls(state='open', direction='asc')
+    return get_items_with_milestone(open_prs)
+
+
+def update_milestone(repo, object_list) -> None:
+    milestone = get_latest_version_milestone(repo)
+    for item in object_list:
+        try:
+            item.edit(milestone=milestone)
```

Comment:
Could it be that in some cases the operation may fail because a REST API call ends with 500, or that it times out? It could be worth it to have a retry wrapper, for the sake of simplicity for the general case anything goes wrong, and only if the issue manifests persistently, propagate an exception upwards.

Reply:
Although possible, I don't remember a 500 type error in this API since I started using it. If an error happens at this point, it should be worth to investigate because it would be related to the objects to be edited. For example, milestone or object deleted while trying to update the item or lack of permission. I don't think it is necessary or useful to have a retry wrapper here.

Reply:
OK, can be added later. However, occasional API failures are generally common when using web services.